### DISCUSSION TITLE
Conditionally hiding a form field based on parent's column value

### DISCUSSION THREAD

**Discussion by robertosanval** [filamentphp/filament#8131] [Help] [answered]
Hello, I want to show or hide a couple of fields on a `RelationManager` based on the value of a parent's record column. Is that possible? How can I get the parent's column value? I've checked the docs, but I need help finding something about this.

**Comment by danharrin** [comment, marked as answer]
`hidden(fn ($livewire) => $livewire->ownerRecord->blablabla)`

**Reply by robertosanval** [reply]
Thanks!
